schema.sql

```diff
@@ -2,9 +2,10 @@
 
 CREATE TABLE animals (
     id INT GENERATED BY DEFAULT AS IDENTITY,
-     name VARCHAR(40),
+     name VARCHAR(100),
      date_of_birth DATE,
      escape_attempts INT,
      neutered BOOLEAN,
-     weight_kg DECIMAL
+     weight_kg DECIMAL,
+     PRIMARY KEY(id)
```

Comment:
- [ ] While inserting data to Animals Table you need to declare neutered as true or false you did correct but except for the first one as you can see in this image 'Agumon' should be neutered. Please fix this issue as well.

<img width="480" alt="Screen Shot 2022-03-28 at 12 31 46 PM" src="https://user-images.githubusercontent.com/69011963/160348939-2bdeae40-2e26-4ebe-9ff8-59e37aa3ca45.png">

Reply:
I have fixed this also. Thank you for noticing.